packages/text-annotator-react/src/TextAnnotatorPopup/TextAnnotatorPopup.tsx

```diff
@@ -138,8 +130,7 @@ export const TextAnnotatorPopup = (props: TextAnnotationPopupProps) => {
           className="a9s-popup r6o-popup annotation-popup r6o-text-popup not-annotatable"
           ref={refs.setFloating}
           style={floatingStyles}
-          {...getFloatingProps()}
-          {...getStopEventsPropagationProps()}>
+          {...getFloatingProps(getStopEventsPropagationProps())}>
```

Comment:
FloatingUI mandates that any custom event listeners should be propagated through the `getReferenceProps`/`getFloatingProps`, [source](https://arc.net/l/quote/cqodxysn): 
> This is because your handler may be either overwritten or overwrite one of the Hooks’ handlers. More event handlers may also be added in future versions.